src/main/res/xml/whereyougo_preferences_directions.xml

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="utf-8"?>
+<PreferenceScreen xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:whereyougo="http://whereyougo"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:icon="@drawable/ic_action_directions"
+    android:key=""
+    android:persistent="false"
+    android:title="@string/pref_guiding">
+
+    <PreferenceCategory
+        android:key="@string/pref_CAT_DIRECTIONS"
+        android:persistent="false"
+        android:title="@string/pref_guiding_compass_sounds">
+
+        <CheckBoxPreference
+            android:defaultValue="@string/pref_DEFAULT_GUIDING_COMPASS_SOUNDS"
+            android:key="@string/pref_KEY_B_GUIDING_COMPASS_SOUNDS"
+            android:summary="@string/pref_guiding_compass_sounds_desc"
+            android:title="@string/pref_guiding_compass_sounds"
+            app:iconSpaceReserved="false" />
+        <ListPreference
+            android:defaultValue="@string/pref_DEFAULT_GUIDING_WAYPOINT_SOUND"
+            android:dependency="@string/pref_KEY_B_GUIDING_COMPASS_SOUNDS"
+            android:entries="@array/pref_guiding_waypoint_sound_entries"
+            android:entryValues="@array/pref_guiding_waypoint_sound_entry_values"
+            android:key="@string/pref_KEY_S_GUIDING_WAYPOINT_SOUND"
+            android:summary="@string/pref_guiding_sound_type_waypoint_desc"
+            android:title="@string/pref_guiding_sound_type"
+            app:iconSpaceReserved="false" />
+        <EditTextPreference
+            android:defaultValue="@string/pref_DEFAULT_GUIDING_WAYPOINT_SOUND_DISTANCE"
+            android:dependency="@string/pref_KEY_B_GUIDING_COMPASS_SOUNDS"
+            android:inputType="number"
+            android:key="@string/pref_KEY_S_GUIDING_WAYPOINT_SOUND_DISTANCE"
+            android:summary="@string/pref_guiding_sound_distance_waypoint_desc"
+            android:title="@string/pref_guiding_sound_distance"
+            whereyougo:previewTemplate="(%1$ m)"
+            app:iconSpaceReserved="false" />
+    </PreferenceCategory>
+
+    <PreferenceCategory
+        android:key=""
```

Comment:
You have added an `android:key` for other `PreferenceCategory`.
If it make sense to do it so then maybe also here?

Reply:
Does make sense. Done via force-push.